backend/src/common/utils.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import threading
+
+from src.db.cache_storage import CacheHandler, RedisStorage
+
+logger = logging.getLogger(__name__)
+
+
+def publish_handler_event(pattern: str, data: dict) -> None:
+    """
+    Publishes an event to a cache storage system.
+
+    Parameters:
+    pattern (str): The pattern of the event.
+    data (dict): The data associated with the event.
+
+    Returns:
+    None
+    """
+    cache_handler = CacheHandler(redis=RedisStorage())
+    logger.info("Start cache storage for publish")
+    cache_handler.publish_event(
+        pattern=pattern,
+        data=data,
+    )
+    logger.info("Cache storage for published to pattern %s data %s", pattern, data)
```

Comment:
THis is wrong, we should inject `cache_handler = CacheHandler(redis=RedisStorage())` through parameters and you should create service cache_handler through constructor in routers, the same way as repository. Im not even sure if 
we should have it in utils
  Example
```
def publish_handler_event(pattern: str, data: dict, cache_handler : ICacheHandler) -> None:
  
    cache_handler.publish_event(
        pattern=pattern,
        data=data,
    )
    logger.info("Cache storage for published to pattern %s data %s", pattern, data)

```